process/block/baseProcess.go

```diff
@@ -1018,7 +1019,7 @@ func (bp *baseProcessor) updateStateStorage(
 	if bytes.Equal(prevRootHash, rootHash) {
 		return
 	}
-
+	log.Info("Got to pruneTrie", "hash", core.ToHex(rootHash))
```

Comment:
No validator cares that the program ran into pruneTrie thing. Might switch this to Debug. Got -> got

Reply:
removed